cvat/apps/annotation/cvat_ddln.py

```diff
@@ -0,0 +1,125 @@
+# Copyright (C) 2020
+
+format_spec = {
+    "name": "DDLN_CSV",
+    "dumpers": [
+        {
+            "display_name": "{name} {format} {version} for images (BB)",
+            "format": "ZIP",
+            "version": "0.1",
+            "handler": "dump_ddln_csv"
+        }
+    ],
+    "loaders": [
+        {
+            "display_name": "{name} {format} {version}",
+            "format": "ZIP",
+            "version": "0.1",
+            "handler": "load",
+        }
+    ],
+}
+
+def extractFileParams(filename):
+    directoryName = filename.split("/")[2]
+    csvFilename = filename.split("/")[-1]
+    csvFilename = csvFilename[:-4] + "_y_bb.csv"
```

Comment:
Will `csv` files be parsed successfully with `_bb` suffix?

Reply:
By whom? If the output is used internally at MSQ side, then you can parse anything. Otherwise, annotation requirements are pretty clear regarding file names.